### PR TITLE
Add changelog format template - bump nuget version for next release - Update nuget tag 

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj [hunk 1]
```diff
@@ -12,9 +12,9 @@
 
   <!-- Nuget properties -->
   <PropertyGroup>
-    <PackageTags>IoT;Digital Twins;$(PackageCommonTags)</PackageTags>
+    <PackageTags>IoT;DigitalTwins;$(PackageCommonTags)</PackageTags>
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj:15]: weird it takes a space as another tag :(

File: sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
\ No newline at end of file
+## 1.0.0-preview.1 (Unreleased)
```
Comment (drwill-ms) [inline sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:3]: Can we fill in more here? Perhaps Sejal can help?

Comment (azabbasi) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:3]: I will ask her

Comment (drwill-ms) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:3]: She's OOF today. I'd add what you think makes sense to start, and she can offer edits when she returns if she has any.

Maybe peruse a few other teams' changelogs for their first release to find prior art?

Comment (azabbasi) [reply sdk/digitaltwins/Azure.DigitalTwins.Core/CHANGELOG.md:3]: Most teams have this initial release log empty. 
I added some template info and an initial log to not have it empty

**Review by drwill-ms — APPROVED** [review]

**Pull request merged**